pingpong-go/src/pingpong/pong/Pong.go

```diff
@@ -1,29 +1,35 @@
-package pong 
+package pong
 
 import (
+	"bytes"
 	"fmt"
 	"net/http"
-	"github.com/gorilla/mux"
 	"strconv"
-	"bytes"
+
+	"github.com/gorilla/mux"
 )
 
 func pongHandler(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	i, _ := strconv.Atoi(params["length"])
+	i, err := strconv.Atoi(params["length"])
+	if err != nil {
+		panic(err)
+	}
 	i -= 4
-	var result bytes.Buffer
-	result.WriteString("po")
-	for ; i > 0; i-- { result.WriteString("o") }
+	result := bytes.NewBufferString("po")
```

Comment:
The type stays the same. The documentation provides following info:
"NewBufferString creates and initializes a new Buffer using string s as its initial contents. It is intended to prepare a buffer to read an existing string. [...]"
Source: https://golang.org/pkg/bytes/#NewBufferString

Therefore this should not affect performance, but makes code more readable.
If wanted I can write a benchmark and compare both options.